doc/BBCode.md

```diff
@@ -1,208 +1,611 @@
-[abstract=apdn]Helly my dear followers on ADN. I made sone new pictures 
-that I wanted to share with you.[abstract]
-Today I was in the woods and took some real cool pictures ...
-</pre>
-
-For Twitter and App.net the system will use the defined abstracts.
-For other networks (e.g. when you are using the "statusnet" connector that is used to post to GNU Social) the general abstract element will be used.
+<table class="bbcodes">
+<tr>
+  <th>BBCode</th>
+  <th>Result</th>
+</tr>
+<tr>
+  <td>
+[abstract]Hi friends Here are my newest pictures![/abstract]<br>
+[abstract=twit]Hi my dear Twitter followers. Do you want to see my new
+pictures?[/abstract]<br>
+[abstract=apdn]Helly my dear followers on ADN. I made sone new pictures
+that I wanted to share with you.[/abstract]<br>
+Today I was in the woods and took some real cool pictures ...</td>
+  <td>For Twitter and App.net the system will use the defined abstracts.<br>
+For other networks (e.g. when you are using the "statusnet" connector that is used to post to GNU Social) the general abstract element will be used.</td>
+</tr>
+</table>
 
 If you use (for example) the "buffer" connector to post to Facebook or Google+ you can use this element to define an abstract for a longer blogpost that you don't want to post completely to these networks.
 
-Networks like Facebook or Google+ aren't length limited. 
-For this reason the [abstract] element isn't used. 
+Networks like Facebook or Google+ aren't length limited.
+For this reason the [abstract] element isn't used.
 Instead you have to name the explicit network:
 
-<pre>
-[abstract]These days I had a strange encounter ...[abstract]
-[abstract=goog]Helly my dear Google+ followers. You have to read my 
-newest blog post![abstract]
-[abstract=face]Hello my Facebook friends. These days happened something 
-really cool.[abstract]
-While taking pictures in the woods I had a really strange encounter ... </pre>
-
-The [abstract] element isn't working with the native OStatus connection or with connectors where we post the HTML. 
-(Like Tumblr, Wordpress or Pump.io)
-
-Special
--------
-
-If you need to put literal bbcode in a message, [noparse], [nobb] or [pre] are used to escape bbcode:
+<table class="bbcodes">
+<tr>
+  <th>BBCode</th>
+  <th>Result</th>
+</tr>
+<tr>
+  <td>
+[abstract]These days I had a strange encounter...[/abstract]<br>
+[abstract=goog]Helly my dear Google+ followers. You have to read my newest blog post![/abstract]<br>
+[abstract=face]Hello my Facebook friends. These days happened something really cool.[/abstract]<br>
+While taking pictures in the woods I had a really strange encounter...</td>
+  <td>Google and Facebook will show the respective abstracts while the other networks will show the default one.<br>
+<br>Meanwhile, Friendica won't show any of the abstracts.</td>
+</tr>
+</table>
+
+The [abstract] element isn't working with the native OStatus connection or with connectors where we post the HTML like Tumblr, Wordpress or Pump.io.
```

Comment:
Can you please add your changes in a separate PR? You seem to have a better idea than me how to rephrase this paragraph since I don't know the ecosystem very well yet.